ipie/addons/ephqmc/hamiltonians/holstein.py

```diff
@@ -0,0 +1,31 @@
+import numpy
+from ipie.hamiltonians.generic_base import GenericBase
+from ipie.utils.backend import arraylib as xp
+
+
+class HolsteinModel(GenericBase):
+    """Class for Holstein model carrying elph tensor and system parameters
+    """
```

Comment:
https://peps.python.org/pep-0008/#documentation-strings : see one-line docstrings

Reply:
I think here would be a good place to provide a detailed description (math) of what the holstein model is, either within the classes docstring or at the module level.